eng/targets/Helix.props

```diff
@@ -12,6 +12,7 @@
   <PropertyGroup>
     <CreateHelixPayload>true</CreateHelixPayload>
     <HelixTimeout>00:30:00</HelixTimeout>
+    <HelixTimeout Condition="$(HelixTargetQueue.StartsWith('Windows.10.Amd64'))">00:40:00</HelixTimeout>
```

Comment:
Why not just make this 40 minutes everywhere?  does 10 minutes make a huge difference?

Reply:
Two reasons
1. I'm not seeing similar timeout errors except on Windows.10.Amd64.Open agents
2. Longer timeouts tie up the agents for longer and make things worse for everybody's jobs

Reply:
How do you know that 10 minutes makes a difference on those agents?

Reply:
Note also this timeout has no effect on the amount of time we wait for the queue, this is how long each workitem has to complete once its on the machine, usually this only gets hit if there's a hang, which is why I ask why we would have a per queue condition on this setting, are the amd64 machines actually timing out due to load/not enough CPU?

Reply:
1. I don't know for sure but we must be fairly close to the current timeout since these tests sometimes succeed.
2. I understand that distinction. In the main #23760 case, the Helix jobs captured DMP files which indicates the timeout was on that end. Steps or jobs that timeout on the regular build agents don't cancel associated Helix runs.

Reply:
Right so my main point is that making this timeout across the board higher, doesn't slow down anyone unless we are hanging, which doesn't sound like the case here

Reply:
My second reason wasn't about the waiting regular build agent but about the Helix agents. Tying them up slows down everyone using the same queues.